cryptoki/src/mechanism/rsa.rs

```diff
@@ -135,12 +135,6 @@ pub struct PkcsPssParams {
     pub s_len: Ulong,
 }
 
-impl From<PkcsPssParams> for Mechanism {
-    fn from(pkcs_pss_params: PkcsPssParams) -> Self {
-        Mechanism::RsaPkcsPss(pkcs_pss_params)
-    }
-}
-
```

Comment:
Good point!

Reply:
This breaks the API... fortunately we're on 0.x and the fix for clients would be that difficult.

Reply:
True, though once you upgrade this conversion simply wouldn't be possible. And I think #68 is a much bigger issue for anyone upgrading :grin:

Reply:
Yes, let's use #68 as an opportunity to clean up stuff and having this From would be super-extremely confusing :)